libs/sdl/sdl.c

```diff
@@ -337,7 +378,11 @@ HL_PRIM SDL_Window *HL_NAME(win_create)(int width, int height) {
 }
 
 HL_PRIM SDL_GLContext HL_NAME(win_get_glcontext)(SDL_Window *win) {
-	return SDL_GL_CreateContext(win);
+	SDL_GLContext ctx = SDL_GL_GetCurrentContext();
+	if (ctx==NULL)
+		ctx = SDL_GL_CreateContext(win);
+	int result = SDL_GL_MakeCurrent(win ,ctx);
+	return ctx;

```

Comment:
I think this is slightly problematic in case we want to have multiple Windows context. We should instead bind SDL_GL_GetCurrentContext and deal with it at the Haxe level (or HL_MOBILE if there's always a single context available?)

Reply:
Fixed in c324c0997fb26089fb1ce749e9ca86643e27467d